spec/javascript/components/supplemental_application/__snapshots__/SupplementalApplicationPage.test.js.snap

```diff
@@ -1662,7 +1662,7 @@ exports[`SupplementalApplicationPage should render the status update dropdown bu
       }
       tabSection={
         Object {
-          "currentUrl": "/",
+          "currentUrl": undefined,
```

Comment:
this doesn't seem like a great change potentially, do you know what caused the change?

Reply:
This change is caused by the fact that we are using `window.location` for navigation and in some cases of mounting the components directly the window object is not consistent. These are the errors where it was saying that the navigation was not implemented. Most of these were with components with form submissions and a reload after the fact